### PR TITLE
fixup nydusctl umount argument usage

### PR DESCRIPTION
**Pull request by cntianci**
## Details
- per [clap document](https://docs.rs/clap/4.3.15/clap/struct.Arg.html#method.index) Arg::index() shouldn’t to be used with [Arg::short](https://docs.rs/clap/4.3.15/clap/struct.Arg.html#method.short) or [Arg::long](https://docs.rs/clap/4.3.15/clap/struct.Arg.html#method.long).
- v5 rafs builder log only chunk info, I guess we miss the inode info.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: builder/src/core/v5.rs [hunk 1]
```diff
@@ -37,6 +37,7 @@ impl Node {
         ctx: &mut BuildContext,
         f_bootstrap: &mut dyn RafsIoWrite,
     ) -> Result<()> {
+        trace!("[{}]\t{}", self.overlay, self);
```
Comment (jiangliu) [inline builder/src/core/v5.rs:40]: Unrelated change?

**Review by imeoer — APPROVED** [review]

**Review by jiangliu — APPROVED** [review]
